src/algorithms/diamondsquare.jl

```diff
@@ -21,48 +23,40 @@ edge midpoints from the nearest two corners and the square's center, where as
 midpoint-displacement only intepolates from the nearest corners (see `MidpointDisplacement`).
 
 """
-struct DiamondSquare <: NeutralLandscapeMaker
-    H::Float64
+@kwdef struct DiamondSquare <: NeutralLandscapeMaker
+    H::Float64 = 0.5
     function DiamondSquare(H::T) where {T <: Real}
         @assert 0 <=  H < 1
         new(H)
     end
 end
 
 """
-    DiamondSquare() 
-
-    Initializes DiamondSqaure with H = 0.5.
-"""
-DiamondSquare() = DiamondSquare(0.5)
+    MidpointDisplacement <: NeutralLandscapeMaker
 
-"""
-    MidpointDisplacement()
+    MidpointDisplacement(; H = 0.5)
 
-    Creates a midpoint-displacement algorithm object `MidpointDisplacement`. The degree of spatial autocorrelation is controlled by a parameter `H`,
-    which varies from 0.0 (low autocorrelation) to 1.0 (high autocorrelation) --- note this is non-inclusive and H = 0 and H = 1 will not behavive as expected.
+Creates a midpoint-displacement algorithm object `MidpointDisplacement`. 
+The degree of spatial autocorrelation is controlled by a parameter `H`,
+which varies from 0.0 (low autocorrelation) to 1.0 (high autocorrelation) --- 
+note this is non-inclusive and H = 0 and H = 1 will not behavive as expected.
 
-    A similar algorithm, diamond-square, functions almost
-    identically, except that in diamond-square, the square step interpolates
-    edge midpoints from the nearest two corners and the square's center, where as 
-    `MidpointDisplacement` only intepolates from the nearest corners (see `DiamondSquare`).
+A similar algorithm, diamond-square, functions almost
+identically, except that in diamond-square, the square step interpolates
+edge midpoints from the nearest two corners and the square's center, where as 
+`MidpointDisplacement` only intepolates from the nearest corners (see `DiamondSquare`).
 """
-struct MidpointDisplacement <: NeutralLandscapeMaker
-    H::Float64
+@kwdef struct MidpointDisplacement <: NeutralLandscapeMaker
+    H::Float64 = 0.5
     function MidpointDisplacement(H::T) where {T <: Real}
         @assert 0 <= H < 1
         new(H)
     end
 end
-MidpointDisplacement() = MidpointDisplacement(0.5)
 
-"""
-    _landscape!(mat, alg::Union{DiamondSquare, MidpointDisplacement}; kw...)
-
-    Check if `mat` is the right size.
-    If mat is not the correct size (DiamondSquare can only run on a lattice of size NxN where N = (2^n)+1 for integer n),
-    allocates the smallest lattice large enough to contain `mat` that can run DiamondSquare.
-"""
+# Check if `mat` is the right size.
+# If mat is not the correct size (DiamondSquare can only run on a lattice of size NxN where N = (2^n)+1 for integer n),
+# allocates the smallest lattice large enough to contain `mat` that can run DiamondSquare.
```

Comment:
Why is this in comments instead of a docstring? I think the docstring versions makes more sense.

Reply:
Docstrings not in docs throw a warning that will be a test failure